src/common/iop_order.c

```diff
@@ -36,6 +35,23 @@
 #define DT_IOP_ORDER_INFO FALSE  // used while debugging
 #define DT_ONTHEFLY_INFO FALSE   // while debugging on-the-fly conversion
 
+/** Note :
+ * we use finite-math-only and fast-math because divisions by zero are manually avoided in the code
```

Comment:
Found the histogram issue. It seems that this first sentence is not correct in fact. Divide by zero is not taken care manually in this part of the code.

Removing "finite-math-only" and "fast-math" from below solves the issue.